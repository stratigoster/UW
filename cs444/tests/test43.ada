PACKAGE foo IS
  procedure boo;
  procedure baz ();
--              ^^
-- syntax error: don't need parentheses if there are no parameters
-- should be: procedure blah;
END foo;
