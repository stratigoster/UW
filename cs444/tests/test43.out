***ERROR***:3:18:	Expected "Id" but found ")"
